hypothesis-python/docs/tutorial/settings.rst

```diff
@@ -66,7 +73,7 @@ To define a setting profile, use |settings.register_profile|:
 
 You can place this code in any file which gets loaded before your tests get run. This includes an ``__init__.py`` file in the test directory or any of the test files themselves. If using pytest, the standard location to place this code is in a ``confest.py`` file (though an ``__init__.py`` or test file will also work).
 
-You can register as many profiles as you want. A profile does not take effect until loaded with |settings.load_profile|:
+Note that a profile does not take effect until loaded with |settings.load_profile|:
```

Comment:
recently changed iirc; we now automatically reload if you define a profile with the current profile name

Reply:
true, but do you think it's worth saying this? Seems like an advanced detail to me

Reply:
We definitely shouldn't say untrue things in the tutorial, even if it's a bit more complicated.

Reply:
that's true. It's pretty tough to work in here for me though. I've tried to sidestep it by rewriting to `a new profile`